src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborWriter.Array.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+using System.Buffers.Binary;
+using System.Text;
+
+namespace System.Security.Cryptography.Encoding.Tests.Cbor
+{
+    internal partial class CborWriter
+    {
+        public void BeginWriteArray(int definiteLength)
```

Comment:
Begin/End feels a lot like the IAsyncResult async pattern.  "Start"/"End" matches the JSON stack, and avoids that legacy taint.

Reply:
I just changed to `WriteStartArray`